HIRS_Utils/src/main/java/hirs/tpm/TPMBaselineGenerator.java

```diff
@@ -641,6 +637,10 @@ private void parseBaseline(final String baselineName, final InputStream inStream
                 TPMBaselineFields.toOSInfo(fieldMap, baseline.getOSInfo()));
             baseline.setTPMInfo(
                 TPMBaselineFields.toTPMInfo(fieldMap, baseline.getTPMInfo()));
+
+            if (baseline.isEmpty()) {
+                throw new TPMBaselineGeneratorException("TPM baseline is empty!");
```

Comment:
Can we add in a test to the `TPMBaselineGeneratorTest` that has an empty CSV with an extra row, as you've suggested to make sure that this is successfully detecting the issue and throwing the expected exception here.